Data/AKC Dog Breeds/main.csv

```diff
@@ -0,0 +1,151 @@
+Breed,height_low_inches,height_high_inches,weight_low_lbs,weight_high_lbs
```

Comment:
Let's remove the data files from version control but include information about how to get them in the respective READMEs.